plugins/airbrake-backend/src/config/ExtractAirbrakeConfig.ts

```diff
@@ -33,9 +34,27 @@ export interface AirbrakeConfig {
  * @public
  *
  * @param config - The config object to extract from
+ * @param logger - THe logger object
  */
-export function extractAirbrakeConfig(config: Config): AirbrakeConfig {
-  return {
-    apiKey: config.getString('airbrake.apiKey'),
-  };
+export function extractAirbrakeConfig(
+  config: Config,
+  logger: winston.Logger,
+): AirbrakeConfig {
+  try {
+    return {
+      apiKey: config.getString('airbrake.apiKey'),
```

Comment:
I was going to suggest that you could use getOptionalString here instead and check if undefined comes out - but I guess that the way you do it here actually leverages the error message that comes out of the config system sensibly :)